### PR TITLE
Add Kitchen tests

### PR DESCRIPTION
**Pull request by n-rodriguez**

### CODE REVIEW

File: pillar.example [hunk 1]
```diff
@@ -1,6 +1,7 @@
+---
 exim:
   config:
-    # See http://pkg-exim4.alioth.debian.org/README/update-exim4.conf.8.html for details.
+    # See http://pkg-exim4.alioth.debian.org/README/update-exim4.conf.8.html for details
```
Comment (myii) [inline pillar.example]: While you didn't add this link, it's currently broken, despite what's said on this page:

* https://wiki.debian.org/PkgExim4

Perhaps we should leave this link but put another one beneath it?

File: docs/README.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/exim-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/exim-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+``exim`` scans log files for malicious activity and executes actions based on what it finds.
```
Comment (myii) [inline docs/README.rst]: @n-rodriguez So you're using `fail2ban-formula` as your "template", huh?!

File: test/integration/default/inspec.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+summary: Verify that the exim formula is setup and configured correctly
+supports:
+  - platform-name: debian
+  - platform-name: ubuntu
+  - platform-name: centos
+  - platform-name: fedora
+  - platform-name: opensuse
+  - platform-name: suse
+  - platform-name: freebsd
+  - platform-name: amazon
```
Comment (myii) [inline test/integration/default/inspec.yml]: Let's remove all of the lines here, except for `debian` and `ubuntu`.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    # - INSTANCE: default-opensuse-leap-15-develop-py3
+    # - INSTANCE: default-amazonlinux-2-develop-py2
+    - INSTANCE: default-debian-9-2019-2-py3
+    - INSTANCE: default-ubuntu-1804-2019-2-py3
+    # - INSTANCE: default-centos-7-2019-2-py3
+    # - INSTANCE: default-fedora-30-2019-2-py3
+    # - INSTANCE: default-opensuse-leap-15-2019-2-py3
+    # - INSTANCE: default-amazonlinux-2-2019-2-py2
+    - INSTANCE: default-debian-9-2018-3-py2
+    - INSTANCE: default-ubuntu-1604-2018-3-py2
```
Comment (myii) [inline .travis.yml]:
```suggestion
    # - INSTANCE: default-ubuntu-1604-2018-3-py2
```

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    # - INSTANCE: default-centos-7-2018-3-py2
+    # - INSTANCE: default-fedora-29-2018-3-py2
+    # - INSTANCE: default-opensuse-leap-15-2018-3-py2
+    # - INSTANCE: default-amazonlinux-2-2018-3-py2
+    - INSTANCE: default-debian-8-2017-7-py2
```
Comment (myii) [inline .travis.yml]:
```suggestion
    # - INSTANCE: default-debian-8-2017-7-py2
```

**Review by myii — CHANGES_REQUESTED** [review]
Thanks @n-rodriguez.  Just a few changes after checking with https://github.com/myii/ssf-formula/pull/37.  I've suggested how to reduce the matrix down to 6 instances as well.  And I've noted that you're not using the `template-formula` as your template!!

**Comment by myii** [comment]
"Closes" #8.

File: docs/README.rst (hunk 2 above)
Comment (n-rodriguez) [reply docs/README.rst]: Ahah! you caught me!

**Review by myii — APPROVED** [review]

**Comment by myii** [comment]
Merged, thanks for all the effort, @n-rodriguez.

**Pull request merged**